saml-core/src/main/java/org/keycloak/saml/common/util/StringUtil.java

```diff
@@ -58,7 +58,21 @@ public static boolean isNotNull(String str) {
     public static boolean isNullOrEmpty(String str) {
         return str == null || str.isEmpty();
     }
+    
+    /**
+     * Convenience method for mapping null, empty (zero length) or whitespace-only strings to null.
+     * 
+     * @param value null or String instance
+     * @return null if value was null or length was zero after whitespace trimming.
+     */
 
+    public static final String nullOrEmptyOrWhitespaceToNull(String value) {
```

Comment:
Please refrain from refactoring irrelevant parts of code. These are scheduled for another task. Kindly remove these changes.